lib/extensions/reset_embroidery_settings.py

```diff
@@ -0,0 +1,47 @@
+import inkex
+
+from .base import InkstitchExtension
+
+
+class ResetEmbroiderySettings(InkstitchExtension):
+    def effect(self):
+        # TODO: Extension does not work for ignored layers/objects
+        if not self.get_elements():
+            return
+
+        for element in self.elements:
+            node = element.node
+            for attrib in node.attrib:
+                if self.selected:
+                    xpath = (".//svg:path[@inkscape:connection-start='#%(id)s' or @inkscape:connection-end='#%(id)s']/parent::*" %
```

Comment:
Check out `commands.find_commands()`, which should find any connected commands for you.